test/demo/deployment.yaml

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demo-deployment
+  labels:
+    app: demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: demo
+  template:
+    metadata:
+      labels:
+        app: demo
+    spec:
+      containers:
+      - name: demo
+        image: docker.io/library/nginx:stable
+        ports:
+        - containerPort: 8080
```

Comment:
This requires configuration of nginx server to listen to port 8080. It listens to port 80 by default, so:

```
--- a/test/demo/deployment.yaml
+++ b/test/demo/deployment.yaml
@@ -21,5 +21,5 @@ spec:
       - name: demo
         image: docker.io/library/nginx:stable
         ports:
-        - containerPort: 8080
+        - containerPort: 80
           name: demo-port
```